### PR TITLE
Improve test coverage of `Plotter.add_floor`

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

This is the follow-up PR of #4865. The purpose of PR is to improve the test coverage of `Plotter.add_floor`.

### Details

- None

### CODE REVIEW

File: pyvista/plotting/renderer.py [hunk 1]
```diff
@@ -1948,7 +1948,7 @@ def add_floor(
             i_size = ranges[2]
             j_size = ranges[1]
         else:
-            raise NotImplementedError(f'Face ({face}) not implementd')
+            raise NotImplementedError(f'Face ({face}) not implemented)
```
Comment (tkoyama010) [inline pyvista/plotting/renderer.py:1951]:
```suggestion
            raise NotImplementedError(f'Face ({face}) not implemented')
```

File: tests/plotting/test_plotter.py [hunk 2]
```diff
@@ -402,3 +402,22 @@ def test_plotter_meshes(sphere, cube):
...
+        ('+Y', (0, -1, 0)),
+        ('+X', (-1, 0, 0)),
+    ],
+)
+def test_plotter_add_floor(face, normal):
+    pl = pv.Plotter()
+    pl.add_floor(face=face)
+    assert np.allclose(pl.renderer._floor.face_normals[0], normal)
+    with pytest.raises(NotImplementedError, match='not implemented'):
+        pl.add_floor(face='invalid')
```
Comment (MatthewFlamm) [inline tests/plotting/test_plotter.py:423]: small comment:  not sure if there is a better way than testing this in every parameterized test.  A failure here will result in 6 failed tests even though it is logically one failure.

**Review by MatthewFlamm — COMMENTED** [review]
An image regression test might be useful, but I think the tests you have here are sufficient for now.  One small comment for consideration, but the current PR state is also fine with me.

File: tests/plotting/test_plotter.py (hunk 2 above)
Comment (tkoyama010) [reply tests/plotting/test_plotter.py:423]: Good point. I was too lazy and skipped separating functions, which was not a good thing.

**Review by MatthewFlamm — APPROVED** [review]
LGTM

**Pull request merged**